adminer-elasticsearch.spec

```diff
@@ -50,6 +50,9 @@ rm -rf $RPM_BUILD_ROOT
 /usr/clearos/sandbox/etc/httpd/conf.d/adminer-elasticsearch.conf
 
 %changelog
+* Mon Jan 07 2019 eGloo <developer@egloo.ca> 4.7.0
```

Comment:
@Manasoft404: you are updating. Why are you putting eGloo?

Reply:
Should I change and put my email address?

Reply:
yes